apps/etterlatte-pdltjenester/src/main/kotlin/person/PersonService.kt

```diff
@@ -64,6 +65,32 @@ class PersonService(
         }
     }
 
+    suspend fun hentPersonNavn(foedselsnummer: Folkeregisteridentifikator): PersonNavn {
+        logger.info("Henter navn for fnr=$foedselsnummer fra PDL")
+
+        // Bruker rolle INNSENDER siden den returnerer minst info om personen
+        val request = HentPersonRequest(foedselsnummer, PersonRolle.INNSENDER, SakType.BARNEPENSJON)
```

Comment:
Dette er midlertidig. Har tenkt å gjøre en større opprydding i separat PR for å standardisere en del av måten vi kaller på PDL i pdltjenester.